tests/acceptance/sudo-test.js

```diff
@@ -0,0 +1,100 @@
+import { click, find, waitFor } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+
+import format from 'date-fns/format';
+
+import { setupApplicationTest } from 'crates-io/tests/helpers';
+
+import { visit } from '../helpers/visit-ignoring-abort';
+
+module('Acceptance | sudo', function (hooks) {
+  setupApplicationTest(hooks);
+
+  const prepare = context => {
```

Comment:
```suggestion
  function prepare(context) {
```

or is there a reason why this needs to be an arrow fn? :)

Reply:
The reason is about ten years of deeply ingrained house styles at multiple companies. :see_no_evil: 

I'll change it.